### ISSUE TITLE
TextBox Background

### ISSUE THREAD

**Issue by angelcalvasp** [wsick/Fayde#191]
Hello everyone 

when you set TextBox Background color you get a border of the color but the Textboxes Background color does not change don't know if I'm doing something wrong

heres the xaml

``` xaml
<Grid x:Name="LayoutRoot" >
        <Grid.DataContext>
            <vms:MainViewModel/>
        </Grid.DataContext>

        <TextBox FontSize="12" BorderThickness="0" Width="100" Height="25" Background="Red" />
    </Grid>
```

heres the output

![untitled](https://cloud.githubusercontent.com/assets/7502746/10598775/8a573fa6-76c1-11e5-89f6-ef987fc81850.png)

**Comment by BSick7** [comment]
This is a bug.  The theme for a TextBox sets the background to `{x:Null}` which is not being obeyed.  Working on a fix now.

**Comment by BSick7** [comment]
Fixed in v0.19.2.

**Comment by angelcalvasp** [comment]
Woah that was fast
